contributed/ra_dec_dc2_run2.2i_dask.ipynb

```diff
@@ -0,0 +1,1455 @@
+    "all_stars[\"g-r\"] = all_stars[\"mag_g\"] - all_stars[\"mag_r\"]\n",
+    "all_stars[\"r-i\"] = all_stars[\"mag_r\"] - all_stars[\"mag_i\"]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "color_color_points = hv.Points(all_stars, kdims=[\"g-r\", \"r-i\"])\n",
+    "color_color = datashade(color_color_points)\n",
+    "\n",
+    "color_mag_points = hv.Points(all_stars, kdims=[\"g-r\", \"mag_r\"])\n",
+    "color_mag = datashade(color_mag_points)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "color_color + color_mag"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are two surprising features of the left plot:\n",
+    "1. The streaks up and to the right.\n",
+    "    These are the finite M-stars models that are then reddened by different amounts of dust, leading to the streaking.  This same reddening affects the other stars as well, but the warmer and hotter stars are distributed along a line parallel to the reddening vector (as an interesting piece of astrophysics that's the subject of a lecture in Astro classes).\n",
+    "2. The curving downward.\n",
+    "    These are various cool white dwarf models, some a bit more theoretical than observed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Crude simple selection of stars in area.\n",
+    "# We're being generous instead of doing the geometry precisely\n",
+    "\n",
+    "min_ra, max_ra = dc2_run2x_wfd[2][0], dc2_run2x_wfd[3][0]\n",
+    "min_dec, max_dec = dc2_run2x_wfd[2][1], dc2_run2x_wfd[0][1]\n",
+    "\n",
+    "stars = all_stars[(min_ra < all_stars[\"ra\"]) & (all_stars[\"ra\"] < max_ra) &\n",
+    "                  (min_dec < all_stars[\"dec\"]) & (all_stars[\"dec\"] < max_dec)]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bright_stars = stars[stars[\"mag_r\"] < 10]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "len(bright_stars)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x, y = pd_moll_ang2xy(bright_stars['ra'], bright_stars['dec'], mollweide=mollweide)\n",
+    "bright_stars_ra_dec = hv.Points((x, y))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We can use the same xticks, yticks <-> RA, Dec mapping as from above.\n",
+    "bright_stars_ra_dec = bright_stars_ra_dec.opts(xlabel='RA', ylabel='Dec',\n",
+    "                                               xticks=major_ticks_and_labels_x, yticks=major_ticks_and_labels_y)"
```

Comment:
I get:

`NameError: name 'major_ticks_and_labels_x' is not defined`

Reply:
Aargh.  Sorry for not ensuring that this Notebook ran clean.